.github/workflows/ci.yml

```
@@ -115,6 +115,38 @@ jobs:
       - name: Test sgx prover
         run: make test
 
+  build-test-sgx-hardware:
+    name: Build and test sgx in hardware
+    runs-on: [self-hosted, sgx, linux]
+    timeout-minutes: 120
+    env:
+      TARGET: sgx
+      CI: 1
+      EDMM: 0
+    steps:
+      - uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - uses: actions-rs/toolchain@v1
+        with:
+          toolchain: stable
+          profile: minimal
+      - name: Install cargo-binstall
+        uses: cargo-bins/cargo-binstall@v1.6.4
+      - name: Install sgx
+        run: TARGET="sgx" CI=1 make install
+      - name: Build sgx prover
+        run: TARGET="sgx" make build
```

```suggestion
        run: make install
      - name: Build sgx prover
        run: make build
```
already set in the env so can skip it here.

---

Fixed